### PR TITLE
Style Events and My Events Page

### PR DESCRIPTION
**Pull request by safchow**
## Trello ticket link

[Developer Task: Implement user feedback for UI/UX of Events Page](https://trello.com/c/vNs8Lr9k/134-developer-task-implement-user-feedback-for-ui-ux-of-events-page)

## Implementation description
* Styled Events and My Events Page
* Styled all dialogue options (View Event, Edit Event, Cancel Event, See Participants)

## Steps to test
1. Verify all dialogue options are working (ex: cancel an event, join an event, verify db is written to)
2. Verify there are no styling inconsistencies
3. Verify correct errors are thrown in error cases

### CODE REVIEW

**Comment by Eshah7** [comment]
Is it possible to make the cards a bit wider?

**Comment by Eshah7** [comment]
Also, I noticed that on my events page, the filter defaults to "Active" - I think we discussed keeping the filter empty for my events, but I do not mind!

**Review by Eshah7 — APPROVED** [review]
lgtm!

File: src/components/events/MyEvents.js
Comment (gabbytang) [inline src/components/events/MyEvents.js:1]: I should have caught this previously, but there seems to be an error with the Edit Events form. I can't change the max participants number. Every other field is able to be updated.

**Review by gabbytang — CHANGES_REQUESTED** [review]
With regards to this implementation, everything looks great, all alerts seem to be working, and the pages are consistent with each other.

There is just one error with the Edit Event form

File: src/components/events/MyEvents.js
Comment (safchow) [reply src/components/events/MyEvents.js:1]: good catch, I put in a fix. Let me know if its resolved

**Comment by safchow** [comment]
> Also, I noticed that on my events page, the filter defaults to "Active" - I think we discussed keeping the filter empty for my events, but I do not mind!

I updated this to keep it consistent with the events page

**Review by gabbytang — CHANGES_REQUESTED** [review]
You fixed the edit participants amount bug. So everything looks perfect except for one small thing. Should update to "edited" 
image

**Review by gabbytang — APPROVED** [review]

**Pull request merged**